client/src/locales/es.ts

```diff
@@ -42,7 +55,7 @@ export default {
 			name: "OpenTogetherTube",
 			text1: "Es una plataforma de sincronizacion de videos en tiempo real.\nEs fácil para usar y no tienes que inscribirte. Simplemente crea un sala, agrega videos y\ninvita a tus amigos. BOOM! Estás listo para disfrutar de videos con tus amigos hasta las 3 de la mañana",
 			text2: "El TogetherTube original fue amado por su interfaz simple,\ny lo fácil que fue empezar a ver vídeos de inmediato.\nOpenTogetherTube pretende ser igual de fácil de usar y luego mejoralo\npara hacerlo aún mejor.",
-			text3: "Actualmente, puedes ver videos en línea con tus amigos de Youtube, Vimeo, enlaces directos a videos .mp4 y",
+			text3: "Actualmente, puedes ver videos en línea con tus amigos de Youtube, Vimeo, Dailymotion, enlaces directos a videos .mp4 y",
```

Comment:
Dailymotion support was disabled in #1670

Reply:
Oops, yea. I pasted this over from a branch that was behind.